unity-renderer/Assets/Rendering/ProceduralSkybox/TimeReporter/TimeReporter.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using DCL;
+using DCL.Interface;
+
+public class TimeReporter : IDisposable
+{
+    private float timeNormalizationFactor;
+    private float cycleTime;
+    private bool wasPaused = true;
+
+    internal event Action<float, bool> OnReport;
+
+    public TimeReporter()
+    {
+        OnReport += DoReport;
+    }
+
+    public void Dispose()
+    {
+        OnReport -= DoReport;
+    }
+
+    public void Configure(float normalizationFactor, float cycle)
+    {
+        timeNormalizationFactor = normalizationFactor;
+        cycleTime = cycle;
+    }
+
+    public void ReportTime(float time)
+    {
+        bool isPaused = !DataStore.i.skyboxConfig.useDynamicSkybox.Get();
```

Comment:
I find this inverse logic hard to follow. I would replace `isPaused` and `wasPaused` for `active` `wasActive` and if changing kernel is bothersome, do the negation in `WebInterface.ReportTime`.

Reply:
done